### PR TITLE
Write simple unit tests for delta and chunk minimizers

### PR DESCRIPTION
**Pull request by mpherman2**

### CODE REVIEW

File: src/python/tests/core/bot/minimizer/chunk_minimizer_test.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the Chunk minimizer."""
+
+import unittest
+
+from bot.minimizer import chunk_minimizer
+
+
+class DeltaMinimizerTest(unittest.TestCase):
```
Comment (mbarbella-chromium) [inline src/python/tests/core/bot/minimizer/chunk_minimizer_test.py]: Nit: update class name

Comment (mpherman2) [reply src/python/tests/core/bot/minimizer/chunk_minimizer_test.py]: Good catch. Done

**Comment by mpherman2** [comment]
I created a base class to use for both tests. Let me know if this is what you had in mind. Thanks

File: src/python/tests/core/bot/minimizer/base_minimizer_tester.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Google LLC
```
Comment (Dor1s) [inline src/python/tests/core/bot/minimizer/base_minimizer_tester.py]: nit: 2020 :)

Comment (Dor1s) [reply src/python/tests/core/bot/minimizer/base_minimizer_tester.py]: and in other files too

Comment (mpherman2) [reply src/python/tests/core/bot/minimizer/base_minimizer_tester.py]: Done

**Pull request merged**